client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -616,7 +644,7 @@ export class CheckoutThankYou extends Component<
 				</Main>
 			);
 		} else if ( wasDomainProduct && ! wasBulkDomainTransfer ) {
-			const [ purchaseType, predicate ] = this.getDomainPurchaseType( purchases );
```

Comment:
Moved this function to `checkout-thank-you/utils` so we can also import it in other components like `<DomainOnlyThankYou>`.